torchani/cuaev/cuaev_cub.cuh

```diff
@@ -1,14 +1,6 @@
 #pragma once
 
-// include cub in a safe manner, see:
-// https://github.com/pytorch/pytorch/pull/55292
-#undef CUB_NS_POSTFIX // undef to avoid redefinition warnings
-#undef CUB_NS_PREFIX
-#define CUB_NS_PREFIX namespace cuaev {
-#define CUB_NS_POSTFIX }
 #include <cub/cub.cuh>
-#undef CUB_NS_POSTFIX
-#undef CUB_NS_PREFIX
```

Comment:
This fails on CUDA 11.5. On the other hand, most torch ops has migrated from thrust to cub, and hopefully there will no longer be namespace conflicting issue.